pkg/events/usermode.go

```diff
@@ -38,12 +39,13 @@ func InitNamespacesEvent() trace.Event {
 	initNamespacesArgs := getInitNamespaceArguments()
 
 	initNamespacesEvent := trace.Event{
```

Comment:
I suggest you turn this to a data source.

Reply:
So for now this is used as state initializer.
This info could be also saved to a datasource.

Reply:
I'm saying we make this a builtin data source, instead of having an event which initializes some shared state (which is what data sources are for). I think this will only be in v0.21 or v0.22, so this can be done as a prep PR.

Reply:
But you will still need an event to initialize the datasource in analyze mode...

Reply:
I had in mind using signals for things like that. In addition to what i've written below, I think --export-analyze should give a separate output of signals, which can be fed back to the control plane running in analyze.

You're right, this needs to be an event or a signal. But, it doesn't need to be an externally visible event. So initialize a data source with this info after the userspace code which would previously emit the event. If running with --export-analyze, dump it as a signal as well.